src/tree/utils/populateTree.ts

```diff
@@ -76,6 +82,13 @@ export const populateTree: Walker = (schema, parent, level, path, options): unde
     } else if (_isObject(node.properties)) {
       (treeNode as TreeListParentNode).children = [];
 
+      if (options?.mergeAllOf && (node.combiner === 'oneOf' || node.combiner === 'anyOf')) {
+        const merged = mergeOneOrAnyOf(fragment, node.combiner);
+        if (merged.length > 0) {
```

Comment:
What is this guard protecting? If merged is empty, should we not set properties to empty?

Reply:
`mergeOneOrAnyOf` will return an empty array if there is no `allOf`, hence the guard.
I'll tweak `mergeOneOrAnyOf` to make sure we always return `node.properties` if there is no `allOf`.